src/Symfony/Component/Form/Extension/Core/Type/IntegerType.php

```diff
@@ -36,9 +37,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $scale = function (Options $options) {
+            if (null !== $options['precision']) {
```

Comment:
This is also what other PRs used that renamed options. Doesn't the key always exists as it has a default?